draft-ietf-tls-wkech.xml

```diff
@@ -68,59 +73,63 @@
   <middle>
     <section title="Introduction">
 
+	<t>
+	    The <xref target="RFC9460">Service Bindings (SVCB) in DNS RFC</xref>
+	    defines how to handle information needed to make connections
+        to services by using a new DNS record set (RRset) that defines
+        the HTTPS and SVCB resource record types.
+        Many origins will have an API that can be used to
+	    dynamically update the DNS RRset to contain the current
+	    list of SVCB data.
+            Some deployments, however, will not, and those deployments could
+            benefit from the mechanism defined here.
+            Note that this protocol is not intended for universal deployment,
```

Comment:
Should this sentence, or similar, be repeated in the abstract section?

Reply:
I didn't do that yet. (Repetition being repetitive but some people do like it in the abstract:-)